llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -23,12 +23,14 @@
 #include "llvm/ADT/Statistic.h"
 #include "llvm/Analysis/MemoryLocation.h"
 #include "llvm/Analysis/VectorUtils.h"
+#include "llvm/CodeGen/ISDOpcodes.h"
```

Comment:
I suppose this is clang-format or clang-tidy adding these as I'm sure you have not introduced the *first* use of anything from these newly added headers.